src/main/java/lotto/game/LottoGame.java

```diff
@@ -0,0 +1,31 @@
+package lotto.game;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import lotto.lotto.Lotto;
+import lotto.shop.Money;
+import lotto.shop.Shop;
+import lotto.ui.Command;
+import lotto.ui.Result;
+
+public class LottoGame {
+    public static void start() {
+        Command command = new Command();
+
+        Shop shop = new Shop();
+        Money money = command.inputMoneyAmount();
+        int amount = shop.buyLotto(money);
+
+        command.printLottoAmount(amount);
+        List<Lotto> lottoBundle = shop.selectAuto(amount);
+        command.printLottoBundle(lottoBundle);
+
+        Set<Integer> answer = command.inputAnswer();
```

Comment:
```suggestion
        LottoNumbers answer = command.inputAnswer();
```
`List<LottoNumber>`를 인스턴스 변수로 갖는 `LottoNumbers`라는 일급 컬렉션을 추가해보면 어떨까요

Reply:
해당 부분도 반영하였습니다. 감사합니다!